policyengine_canada/tests/gov/cra/tax/income/credits/gst_credit/gst_credit_singles_boost.yaml

```diff
@@ -0,0 +1,42 @@
+- name: Single person with income 15,000 gets a boost of (15,000 - 9919) * 0.02 = 101.62
+  period: 2023
+  input:
+    gst_credit_category: HEAD
```

Comment:
Also below - unit tests should only have `is_married`, `gst_credit_single_parent_household`, and `household_net_income` as inputs
```suggestion
```